patterns/1-initial/introducing-metrics-in-innersource.md

```diff
@@ -89,8 +102,11 @@ Initial
 - Clint
 - Russ Rutledge
 - Tom
+- Jack Yang
 
 ## Acknowledgement
 
 - Georg
 - Bob
+- [Aaron Stewart](https://github.com/a-a-ron/innersource-template-pluralsight/tree/master/metrics)
+- Wilson Mar
```

Comment:
@arlou do you have any online resource by Wilson Mar that you want to reference here?

Reply:
hi @spier, thanks for checking; So far, no referred link on Wilson as I referred to his inputs from Slack channel.